tools/ci_build/github/azure-pipelines/orttraining-win-gpu-ci-pipeline.yml

```diff
@@ -3,7 +3,7 @@ trigger: none
 jobs:
 - template: templates/win-ci-2019.yml
   parameters:
-    AgentPool : 'Win-GPU-2019'
+    AgentPool : 'onnxruntime-win-cuda11-0'
```

Comment:
The "Win-GPU-2019" machine pool doesn't have CUDA 11.0 anymore. It was removed when I added CUDA 11.1 in, to make room for the new one. 

So I created a new pool under name of "onnxruntime-win-cuda11-0" for this PR.